cmd/operator-sdk/run/cmd.go

```diff
@@ -74,16 +73,24 @@ func NewCmd() *cobra.Command {
 					log.Fatalf("Failed to run operator using OLM: %v", err)
 				}
 			case c.local:
+				// set --watch-namespace flag if the --namespace flag is set
+				// (only if --watch-namespace flag is not set)
+				if cmd.Flags().Changed("namespace") {
+					log.Info("--namespace is deprecated; use --watch-namespace instead.")
```

Comment:
👍 
```suggestion
					log.Info("--namespace is deprecated; use --watch-namespace and/or --operator-namespace instead.")
```

Reply:
in `operator-sdk run` command the equivalent flag for old `--namespace` is `--watch-namespace`.
There is an already existing flag `--operator-namespace` in `operator-sdk run --local` command.

In fact. it is this `--operator-namespace` flag which made me adopt the name `--operator-namespace` in the `operator-sdk test --up-local` (instead of `--deploy-namespace` we planned in the initial discussions.